### PR TITLE
chore(TDI-45395): bump component-runtime to 1.28.0

### PR DESCRIPTION
**Pull request by undx**
https://jira.talendforge.org/browse/TDI-45395

Also
- Upgrade log4j to 2.14.0
- Upgrade beam to 2.26.0
- Upgrade junit5 to 5.70.0
- Upgrade lombok to 1.18.16

### CODE REVIEW

**Review by clesaec — COMMENTED** [review]
Can you put a link to the related JIRA in comment ?

**Review by ypiel-talend — APPROVED** [review]
LGTM

File: rest/src/test/java/org/talend/components/rest/service/ClientTestWithMockProxyTest.java [hunk 1]
```diff
@@ -258,7 +257,7 @@ void testFactsWithTRACE() {
         assertEquals(5.0d, record.getDouble("upvotes"));
     }
 
-    @EnvironmentalTest
+    // @EnvironmentalTest
```
[the remark this answers is not in the document]
Comment (undx) [reply rest/src/test/java/org/talend/components/rest/service/ClientTestWithMockProxyTest.java:260]: yep, explanation made in daily

**Review by clesaec — APPROVED** [review]
:+1:

**Review by ozhelezniak-talend — APPROVED** [review]
LGTM

**Pull request merged**